view/adminhtml/web/js/validation.js

```diff
@@ -0,0 +1,44 @@
+require([
+        'jquery',
+        'Magento_Ui/js/modal/alert',
+        'domReady!'
+    ],
+    function($, alert){
+        var apiMode = $("#payment_us_breadcheckout_api_mode").val(),
+            prodKey = "#payment_us_breadcheckout_api_public_key",
+            prodSecret = "#payment_us_breadcheckout_api_secret_key",
+            sandKey = "#payment_us_breadcheckout_api_sandbox_public_key",
+            sandSecret = "#payment_us_breadcheckout_api_sandbox_secret_key",
+            selector = [prodKey,prodSecret,sandKey,sandSecret].join(", "),
+            validationUrl = window.location.origin + "/admin/breadadmin/bread/validateCredentials";
+
+        $(selector).on("input",function () {
+
+            var key = apiMode === "1" ? $(prodKey).val() : $(sandKey).val();
+            var secret = apiMode === "1" ? $(prodSecret).val() : $(sandSecret).val();
+
+            if(secret.indexOf('*') === -1){
```

Comment:
Could you break this condition out into a different variable? just to give it a clear name to explain what that logic is meant to check for, since in the future it might not be clear the significance of * being present or not

Reply:
yes